src/sentry/integrations/slack/utils.py

```diff
@@ -159,6 +153,8 @@ def build_group_attachment(group, event=None, tags=None, identity=None, actions=
 
     members = get_member_assignees(group)
     teams = get_team_assignees(group)
+    if event is None:
+        event = group.get_latest_event()
```

Comment:
Nit: I don't know that we need to do this? The group title should just reflect the latest event. Not blocking on this

Reply:
Removed that line.

Reply:
Added in two tests. A call to build_group_attachment with a group to get back group info, and with a group and event to get back event info.